examples/quickstart/jupyter-notebooks/sql-tutorial.ipynb

```diff
@@ -0,0 +1,765 @@
+    "print(f\"\\033[1mQuery endpoint\\033[0m: {druid_host+endpoint}\")\n",
+    "http_method = \"POST\"\n",
+    "\n",
+    "payload = json.dumps({\n",
+    "  \"query\":\"SELECT TIME_IN_INTERVAL(__time, '2016-06-27T12:00:00/2016-06-27T13:00:00'), COUNT(*) FROM \\\"wikipedia-sql-tutorial\\\" \\\n",
+    "  GROUP BY TIME_IN_INTERVAL(__time, '2016-06-27T12:00:00/2016-06-27T13:00:00')\"\n",
+    "})\n",
+    "headers = {'Content-Type': 'application/json'}\n",
+    "\n",
+    "response = requests.request(http_method, druid_host+endpoint, headers=headers, data=payload)\n",
+    "\n",
+    "print(\"\\033[1mQuery\\033[0m:\\n\" + payload)\n",
+    "print(\"\\n\\033[1mResponse\\033[0m: \\n\" + json.dumps(response.json(), indent=4))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f7cfdfae-ccba-49ba-a70f-63d0bd3527b2",
+   "metadata": {},
+   "source": [
+    "### NULL values\n",
+    "\n",
+    "Druid supports SQL compatible NULL handling, allowing string columns to distinguish empty strings from NULL and numeric columns to contain NULL rows. To store and query data in SQL compatible mode, explicitly set the `useDefaultValueForNull` property to `false` in `_common/common.runtime.properties`. See [Configuration reference](https://druid.apache.org/docs/latest/configuration/index.html) for common configuration properties.\n",
+    "\n",
+    "When `useDefaultValueForNull` is set to `true` (default behavior), Druid stores NULL values as `0` for numeric columns and as `''` for string columns."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "29c24856",
+   "metadata": {
+    "deletable": true,
+    "tags": []
+   },
+   "source": [
+    "## SELECT statement syntax\n",
+    "\n",
+    "Druid SQL supports SELECT statements with the following structure:\n",
+    "\n",
+    "``` mysql\n",
+    "[ EXPLAIN PLAN FOR ]\n",
+    "[ WITH tableName [ ( column1, column2, ... ) ] AS ( query ) ]\n",
+    "SELECT [ ALL | DISTINCT ] { * | exprs }\n",
+    "FROM { <table> | (<subquery>) | <o1> [ INNER | LEFT ] JOIN <o2> ON condition }\n",
+    "[ WHERE expr ]\n",
+    "[ GROUP BY [ exprs | GROUPING SETS ( (exprs), ... ) | ROLLUP (exprs) | CUBE (exprs) ] ]\n",
+    "[ HAVING expr ]\n",
+    "[ ORDER BY expr [ ASC | DESC ], expr [ ASC | DESC ], ... ]\n",
+    "[ LIMIT limit ]\n",
+    "[ OFFSET offset ]\n",
+    "[ UNION ALL <another query> ]\n",
+    "```\n",
+    "\n",
+    "As a general rule, use the LIMIT clause with `SELECT *` to limit the number of rows returned. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8cf212e5-fb3f-4206-acdd-46ef1da327ab",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## WHERE clause\n",
+    "\n",
+    "Druid SQL uses the [SQL WHERE clause](https://druid.apache.org/docs/latest/querying/sql.html#where) of a SELECT statement to fetch data based on a particular condition.\n",
+    "\n",
+    "In most cases, specifying a time range when using the WHERE clause improves query performance.\n",
```

Comment:
Is it worth mentioning that it is also useful to include the dimensions used in partitioning? You can say that it is covered later.